### PR TITLE
Updates due to changes in oe core

### PR DESCRIPTION
**Pull request by cshoredaniel**
OE-Core has become more stringent and catches more bugs, so we need to update some recipes to be able to build at all (and along the way we fix some nice things like a buffer overflow.

### CODE REVIEW

File: recipes-tweaks/busybox/busybox_append.inc [hunk 1]
```diff
@@ -3,12 +3,11 @@
 
 # Released under the MIT license (see COPYING.MIT for the terms)
 
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```
Comment (kraj) [inline recipes-tweaks/busybox/busybox_append.inc]: lets not intoduce another .inc to maintain, having a bbappend is simple enough here.

Comment (cshoredaniel) [reply recipes-tweaks/busybox/busybox_append.inc]: k, was thinking of non-master oe-core, but I guess that's not relevant, really.

Comment (kraj) [reply recipes-tweaks/busybox/busybox_append.inc]: yes eventually we will branch when sumo( 2.5) branches out, for now its just master

**Comment by cshoredaniel** [comment]
ok, should be good now.

**Pull request merged**